web/packages/teleterm/src/ui/DocumentCluster/resourcesContext.tsx

```diff
@@ -51,24 +57,24 @@ export const ResourcesContextProvider: FC<PropsWithChildren> = props => {
     emitterRef.current = new EventEmitter();
   }
 
-  // This function could be expanded to emit a cluster URI so that a request refresh for a root
-  // cluster doesn't trigger refreshes of leaf DocumentCluster instances and vice versa.
-  // However, the implementation should be good enough for now since it's used only in Connect My
-  // Computer setup anyway.
   const requestResourcesRefresh = useCallback(
-    () => emitterRef.current.emit('refresh'),
+    (rootClusterUri: RootClusterUri) =>
```

Comment:
What do you think about making the hook accept the `rootClusterUri` argument? Then the caller doesn't need to remember to pass it to every function that the hook provides.

Reply:
Makes sense, done.